### DISCUSSION TITLE
.umi/plugin-tailwindcss/tailwind.css does not generate in monorepo

### DISCUSSION THREAD

**Discussion by kun391** [umijs/umi#10161] [Q&A] [answered]
I'm using Turborepo for my project. 

When I run dev on the app the `.umi/plugin-tailwindcss/tailwind.css` was generated in `.umi`
But it does not generate when I run in root project like `turbo run dev --filter @app`

Any idea on this? Is it a bug or I'm missing something?

Thanks!

**Comment by fz6m** [comment, marked as answer]
work for me, please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example)

**Reply by kun391** [reply]
[point-minimal.zip](https://github.com/umijs/umi/files/10317216/point-minimal.zip)


At the root, you can use `pnpm i` then run the app by `pnpm dev`
The issue occurs at root of the project.

<img width="460" alt="Screenshot 2022-12-29 at 08 42 55" src="https://user-images.githubusercontent.com/5398707/209892651-703aed97-da14-4ed7-a9b2-d0a5151889ba.png">


When I run `pnpm dev` in `apps/front`. It works fine

<img width="470" alt="Screenshot 2022-12-29 at 08 44 36" src="https://user-images.githubusercontent.com/5398707/209892734-c314b35d-7d29-4f40-a70b-e2acfe498112.png">

Could you please have see it @fz6m? Thanks!

**Reply by fz6m** [reply]
See https://github.com/tailwindlabs/tailwindcss/pull/9331

You should use tailwind `3.1.8` version or use `concurrently` start tailwind spawn.

```ts
// package.json
  "tailwindcss": "3.1.8"
```

or

```bash
  pnpm add -D concurrently
```

```ts
// package.json
  "dev": "concurrently \"umi dev\" \"tailwindcss -i ./tailwind.css -o ./src/.umi/plugin-tailwindcss/tailwind.css -c ./tailwind.config.js --watch\""
```

FYI :

 - https://github.com/tailwindlabs/tailwindcss/issues/9264#issuecomment-1247248235

 - https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.2.0

**Reply by kun391** [reply]
Awesome, Its works for me. 

Thank you so much, @fz6m
